### PR TITLE
Allow pvc to be used. Add a kyverno exception

### PR DESCRIPTION
**Pull request by allanger**
Zot requires a PVC and gets blocked by Kyverno

### CODE REVIEW

File: helm/zot/templates/pvc.yaml [hunk 1]
```diff
@@ -13,3 +13,30 @@ spec:
...
   {{- end }}
 {{- end }}
+
+{{- if or (.Capabilities.APIVersions.Has "kyverno.io/v2alpha1/PolicyException") (eq .Values.pvc.policyException.enforce true) }}
+apiVersion: kyverno.io/v2alpha1
+kind: PolicyException
+metadata:
+  name: allow-pvc-{{ .Release.Name | quote }}
+  {{- if (.Values.pvc.policyException).namespace }}
+  namespace: {{ .Values.pvc.policyException.namespace }}
```
Comment (uvegla) [inline helm/zot/templates/pvc.yaml:23]: I think you forgot to add it to the values with a default. I think there are some restrictions at least on MCs where policy exceptions can be installed. Schema needs to be regenerated as well probably.

Comment (uvegla) [reply helm/zot/templates/pvc.yaml:23]: Thinking about this: https://gigantic.slack.com/archives/C02FL4EAADD/p1706793331744919

**Review by uvegla — CHANGES_REQUESTED** [review]
Comment + schema needs to be regenerated I think.

File: helm/zot/templates/pvc.yaml (hunk 1 above)
Comment (allanger) [reply helm/zot/templates/pvc.yaml:23]: it should be fixed

Comment (uvegla) [reply helm/zot/templates/pvc.yaml:23]: Thanks! LGMT! 👍🏻

**Review by uvegla — APPROVED** [review]

**Pull request merged**